R/pkg/R/DataFrame.R

```diff
@@ -2401,4 +2401,41 @@ setMethod("str",
                 cat(paste0("\nDisplaying first ", ncol(localDF), " columns only."))
               }
             }
-          })
\ No newline at end of file
+          })
+
+#' Saves the content of the DataFrame to an external database table via JDBC
+#'
+#' Additional JDBC database connection properties can be set (...)
+#'
+#' Also, mode is used to specify the behavior of the save operation when
+#' data already exists in the data source. There are four modes: \cr
```

Comment:
yes, it forces a new line in the generated doc, otherwise roxygen2 removes new line